app/test/test_datastore.py

```diff
@@ -0,0 +1,383 @@
+
+    @mock_dynamodb2
+    def test_list_items_sessions(self):
+        table = self.create_session_table()
+        response = table.scan()
+        items = response['Items'] if 'Items' in response else {}
+        self.assertEquals(len(items), 0)
+
+
+        d = DataStore('dynamodb', 'Sessions', 'PatientId', 'SessionId')
+        info = {DATASTORE_COLUMN_SESSION_ID : 's-1',
+                DATASTORE_COLUMN_PATIENT_ID: 'p-1',
+                DATASTORE_COLUMN_HEALTH_CARE_PROFESSSIONAL_ID: 'h-1',
+                DATASTORE_COLUMN_SESSION_NAME: 'SessionName',
+                DATASTORE_COLUMN_COMPREHEND_S3_PATH: 'ComprehendS3Path',
+                DATASTORE_COLUMN_TRANSCRIBE_S3_PATH: 'TranscribeS3Path',
+                DATASTORE_COLUMN_TIMESTAMP_START: 1969,
+                DATASTORE_COLUMN_TIMESTAMP_END: 2030}
+        response = d.save(info, 'us-west-2')
+        response = table.scan()
+        items = response['Items'] if 'Items' in response else {}
+        self.assertEquals(len(items), 1)
+        for item in items:
+            if 's-1' in item[DATASTORE_COLUMN_SESSION_ID]:
+                self.assertEquals('p-1', item[DATASTORE_COLUMN_PATIENT_ID])
+
+        info = {DATASTORE_COLUMN_SESSION_ID : 's-2',
+                DATASTORE_COLUMN_PATIENT_ID: 'p-2',
+                DATASTORE_COLUMN_HEALTH_CARE_PROFESSSIONAL_ID: 'h-2',
+                DATASTORE_COLUMN_SESSION_NAME: 'SessionName',
+                DATASTORE_COLUMN_COMPREHEND_S3_PATH: 'ComprehendS3Path',
+                DATASTORE_COLUMN_TRANSCRIBE_S3_PATH: 'TranscribeS3Path',
+                DATASTORE_COLUMN_TIMESTAMP_START: 1969,
+                DATASTORE_COLUMN_TIMESTAMP_END: 2030}
+        response = d.save(info, 'us-west-2')
+        response = table.scan()
+        items = response['Items'] if 'Items' in response else {}
+        self.assertEquals(len(items), 2)
+        for item in items:
+            if 's-2' in item[DATASTORE_COLUMN_SESSION_ID]:
+                self.assertEquals('p-2', item[DATASTORE_COLUMN_PATIENT_ID])
+
+        info = {DATASTORE_COLUMN_SESSION_ID : 's-3',
+                DATASTORE_COLUMN_PATIENT_ID: 'p-3',
+                DATASTORE_COLUMN_HEALTH_CARE_PROFESSSIONAL_ID: 'h-3',
+                DATASTORE_COLUMN_SESSION_NAME: 'SessionName',
+                DATASTORE_COLUMN_COMPREHEND_S3_PATH: 'ComprehendS3Path',
+                DATASTORE_COLUMN_TRANSCRIBE_S3_PATH: 'TranscribeS3Path',
+                DATASTORE_COLUMN_TIMESTAMP_START: 1969,
+                DATASTORE_COLUMN_TIMESTAMP_END: 2030}
+        response = d.save(info, 'us-west-2')
+        response = table.scan()
+        items = response['Items'] if 'Items' in response else {}
+        self.assertEquals(len(items), 3)
+        for item in items:
+            if 's-3' in item[DATASTORE_COLUMN_SESSION_ID]:
+                self.assertEquals('p-3', item[DATASTORE_COLUMN_PATIENT_ID])
+
+    @mock_dynamodb2
+    def test_list_items_patients(self):
```

Comment:
Similar comments to those mentioned on "test_list_items_sessions"

Reply:
Done.